tg-portfolio/components/about/Services.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+
+const Services = () => (
+  <div className="mt-10 md:mx-24 sm:mx-5 ">
+    <div className="p-6 rounded-lg bg-neutral/60 animate-in slide-in-from-bottom-48 duration-1000">
+      <h1 className="test text-4xl text-black font-bold border-green-700/100 pb-8 px-8 border-b-4">Services</h1>
+      <div className="flex flex-auto justify-between sm:gap-5 flex-row p-5 overflow-auto">
+        <div className="flex flex-row items-center">
+          <Image src="/icons8-stack-64.png" width={65} height={65} />
+          <h2 className=" font-bold">Full Stack Development</h2>
+        </div>
+        <div className="flex flex-row items-center">
+          <p className="font-extrabold hidden md:flex pl-5 text-black"> | | </p>
+        </div>
```

Comment:
**This structure is getting repeated in the very next lines. We can refactor this code.** 
   
Create an array of object and map over this structure:

```javascript
const services = [
   {
      id: "service-1",
      title: "Full Stack Development"
   },
   ...
]

{services.map((service) => (
  ...structure
))}
```

Reply:
Thanks!!!